client/extensions/woocommerce/app/settings/email/mailchimp/setup-mailchimp.js

```diff
@@ -296,12 +304,12 @@ class MailChimpSetup extends React.Component {
 					<div className="mailchimp__setup-dialog-title">MailChimp</div>
 					<ProgressBar
 						value={ stepNum + 1 }
-						total={ Object.keys( steps ).length }
+						total={ Object.keys( steps ).length - 1 }
```

Comment:
Suggestion - since step values are re-used (i.e. 2) - define a const up top near the steps with the last step number and use that for total instead.  That way you can continue to re-use values if desired without having to tweak the subtraction here.